spec/controllers/relationships_controller_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'spec_helper'
+
+describe RelationshipsController do
+
+  describe 'GET #index' do
+    context "with unauthorized user" do
+      it_behaves_like 'require sign in' do
+        let(:action) { get :index }
+      end
+    end
+
+    context "with authorized user" do
+      login_user
+
+      it "sets @relationships with collection of relationships initiated by signed in user" do
+        user1 = create(:user)
+        user2 = create(:user)
+        rel1 = create(:relationship, follower: current_user, followed_id: user1)
```

Comment:
Confusing part for me is this `followed_id: user1` you want `id`, but you pass in object.